module/userProjects/helper.js

```diff
@@ -1944,15 +1944,15 @@ module.exports = class UserProjectsHelper {
 				if (UTILS.revertStatusorNot(appVersion)) {
 					projectDocument.status = UTILS.revertProjectStatus(projectDocument.status)
 				}
-				let response = await reportService.projectAndTaskReport(userToken, projectDocument, projectPdf)
 
-				if (response && response.success == true) {
+				let response = await common_handler.unnatiViewFullReportPdfGeneration(projectDocument, userToken)
```

Comment:
@MallanagoudaB Why are you passing user token instead user id? I have added this comment in prev review also